### ISSUE TITLE
Add workspace name to the window title

### ISSUE THREAD

**Issue by natenho** [a85/POSTMan-Chrome-Extension#6291]
When opening two or more Postman windows to work on different workspaces, the taskbar shows only "Postman" in the window caption, so it's confusing to switch to the right window.

**Describe the solution you'd like**
Add the workspace name in the postman window caption or something to differentiate between windows when switching between them in the task bar.

**Comment by jolsen71** [comment]
I came here for this exact same request. It is also an issue when trying to find the right Postman window to share in meetings, since I have 3 different windows open at once in different workspaces because that is generally (with the exception of this specific issue) much easier than attempting to flip back and forth between workspaces, even when they're favorited. I'm disappointed that this task that should require fairly minimal effort is still an open request almost 2 years later. :(

Which window is for which workspace? It currently requires you to bring each one to the foreground/focus until you find the one you want. Let's say I have three windows open, one for Workspace A, one for Workspace B, and one for Workspace C. 


Windows open on desktop - which window is Workspace C?
![image](https://user-images.githubusercontent.com/6641548/109572531-01087c00-7ab3-11eb-8179-5fcf80a40be3.png)

.

Taskbar - Which window is Workspace A? Keep in mind the order on the taskbar can change as you drag windows from monitor to monitor.
![image](https://user-images.githubusercontent.com/6641548/109572551-11205b80-7ab3-11eb-9f35-814dee9e7f56.png)

.

Choose a window to share in Teams - which one is Workspace B?
![image](https://user-images.githubusercontent.com/6641548/109574136-0666c600-7ab5-11eb-9186-5362b6bebf20.png)

**Comment by jolsen71** [comment]
### Feature request add-on 1:

Please also include the selected environment in the title. I now find myself having two windows both open to the same workspace but with different environments selected, so I can easily interactively compare the results of an endpoint call between environments.

### Feature request add-on 2:

It would be even nicer to be able to add a short bit of custom text per window title too, which I know is unlikely to happen. But if it were a feature, then I could literally name/title each window to indicate what I am specifically using that window for. So if I were helping Mike, I could set the configurable string to "Helping Mike" and it would appear in the window title along with everything else.

### Feature request add-on 3 - bonus round:

Maybe the whole window title could be a configurable string.

With a template string something like
`WS: {workspacetype} {workspacename} - ENV: {envname} - {customtext}`

and with me helping Mike troubleshoot something on ExportAPI on QA, and if I set the custom text to "Helping Mike", the Window title would appear like:
`Postman - WS: Team ExportAPI - ENV: QA - Helping Mike`

### In summary

On really busy days when I'm bouncing around and helping a number of different people, I find myself with 4 and even up to 8 postman windows open and it would be great to be able to more quickly tell them apart by the taskbar/window title. 

Having all of this would be AWESOME to me.

Thanks for listening.

**Comment by chops888** [comment]
Came for the same thing, for all of the above reasons, AND for automatic time-tracking software. One collection or workspace is usually assoc'd with a particular client/project. Would help tremendously for billing purposes.

**Comment by jeroen-vanderpoll** [comment]
Hi @natenho , thanks for raising this.

If you update to the most recent version of Postman, all windows now include the workspace name in the title. We hope this helps with your use case. Let us know if you need anything else.

<img width="706" height="152" alt="Image" src="https://github.com/user-attachments/assets/c8379e99-a75b-4608-b63e-6563fb05409f" />
